.github/workflows/rpc-parity.yml

```diff
@@ -0,0 +1,58 @@
+name: RPC parity tests
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+  pull_request:
+    branches:
+      - main
+jobs:
+  rpc-parity:
+    name: RPC parity tests
+    runs-on: ubuntu-24.04
+    steps:
+      # Remove some unnecessary software to free up space. This should free up around 15-20 GB.
+      # This is required because of the limited space on the runner,
+      # and disk space-hungry snapshots used in the setup.
+      # This is taken from:
+      # https://github.com/easimon/maximize-build-space/blob/fc881a613ad2a34aca9c9624518214ebc21dfc0c/action.yml#L121-L136
+      # Using the action directly is not feasible as it does some more modifications that break the setup in our case.
+      - name: Remove unnecessary software
+        run: |
+            echo "Disk space before cleanup"
+            df -h
+            sudo rm -rf /usr/share/dotnet
+            sudo rm -rf /usr/local/lib/android
+            sudo rm -rf /opt/ghc
+            sudo rm -rf /opt/hostedtoolcache/CodeQL
+            echo "Disk space after cleanup"
+            df -h
+      - uses: actions/checkout@v4
+      - name: Run api compare tests on calibnet
+        if: false
```

Comment:
That's one way to ensure the checkmark is always green, but perhaps we want to run the test from time to time? :rofl:

Reply:
Sorry I was testing workflow url in issue template